t/unit-tests/t-reftable-tree.c

```diff
@@ -44,13 +43,29 @@ static void test_tree(void)
 		check(nodes[i] == tree_search(values + i, &root, &test_compare, 0));
 	}
 
-	infix_walk(root, check_increasing, &c);
+	tree_free(root);
+}
+
+static void test_infix_walk(void)
+{
+	struct tree_node *root = NULL;
+
```

Comment:
Same here, let's drop this useless newline and instead add it after the variable declarations.